.github/actions/generate-commit-list-v1/src/run.test.ts

```diff
@@ -49,17 +47,17 @@ describe('run', () => {
 
       assert.isTrue(setFailed.calledOnce)
       assert.isTrue(
-        setFailed.calledWith('Input required and not supplied: base')
+        setFailed.calledWith(
+          'You must provide either a tag or both a base and head branch.'
+        )
       )
     })
   })
 
   describe('when the head input is not provided', () => {
     it('should throw an error', async () => {
       getInput.withArgs('base', { required: true }).returns('main')
```

Comment:
`required: true` is no longer passed so the stub isn't ever called. This still passed the test though as each of the inputs were missing, but failing for the wrong reason.

```suggestion
      getInput.withArgs('base').returns('main')
```

Reply:
This still needs to be corrected as the stub is not being called as `{ required: true}` is not passed. This means it's failing because the `base` input is not being set and not because the `head` input.